r/R/schema.R

```diff
@@ -328,3 +339,7 @@ print.arrow_r_metadata <- function(x, ...) {
   utils::str(.unserialize_arrow_r_metadata(x))
   invisible(x)
 }
+
+schema_check <- function(items) {
+  all(map_lgl(.list, ~ inherits(.x, "Field"))) || FALSE
+}
```

Comment:
Is `schema_check` used elsewhere? I don't see it but I might be missing something?

Reply:
Oops, intended to use it and then didn't, will update

Reply:
It might be helpful as a helper especially if we accept a list as input (or for testing). But up to you if you think it's better as a helper or inline

Reply:
Had to refactor now so dropped it as it no longer works